taylorseager/CulinaryChronicles
# Finishing create recipe form

## Description
Finalized create recipe form; favorite toggle and radio buttons working, category dropdown pulling through

## Related Issue
#11 

## Motivation and Context
User needs to have a working form to add new data entries.

## How Can This Be Tested?
Pull and checkout branch, run local server
got to create recipe tab
fill out fields
once submit is selected, it should route to recipes page

## Types of changes
